python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -186,6 +186,7 @@ def __init__(
         config_reader: Union[str, Callable[[], dict]],
         load_metrics: LoadMetrics,
         gcs_node_info_stub: gcs_service_pb2_grpc.NodeInfoGcsServiceStub,
+        session_name: str,
```

Comment:
Optional[str]?
Alternatively, have the get_session_name return an empty string to keep a str type.